spec/requests/stylesheets_spec.rb

```diff
@@ -1,6 +1,6 @@
 require 'rails_helper'
 
-RSpec.describe "Stylesheets", type: :request do
+RSpec.describe "Stylesheets", type: :request, unless: ActiveAdmin.application.use_webpacker do
```

Comment:
Very cheeky. I didn't know we could skip a spec like this. 👌🏻

Any value in having a similar spec like this for webpacker, if we don't already? My guess is no but figured I'd ask.

Reply:
I'm pretty sure it would be too late to test for css errors in rails specs :) any error should appear when bundling with rollup anyway